### ISSUE TITLE
Super linter not work on Docker and Azure DevOps for C# language

### ISSUE THREAD

**Issue by andreatosato** [github/super-linter#3048]
Hi,
I have one empty solution with some sample files.
I attach solution.
I copy .editorconfing in root folder (for docker purpose) and in src folder (for visual studio purpose).

I run this command on docker and on Azure Devops but I don't receive warning for LinterTest.

```bash
docker run `
-e RUN_LOCAL=true `
-e ACTIONS_RUNNER_DEBUG=true `
-e LINTER_RULES_PATH='.' `
-e EDITORCONFIG_FILE_NAME='.editorconfig' `
-e VALIDATE_CSHARP=true `
-e VALIDATE_ALL_CODEBASE=false `
-e DEFAULT_BRANCH=master `
-v D:\DevOps\phyd\AEP.API.ESCONet.Customization.WebPortal:/tmp/lint `
-v D:\DevOps\phyd\AEP.API.ESCONet.Customization.WebPortal\.editorconfig:/tmp/lint/.editorconfig `
 github/super-
[AEP.API.ESCONet.Customization.WebPortal.zip](https://github.com/github/super-linter/files/8939782/AEP.API.ESCONet.Customization.WebPortal.zip)
linter:latest
```

If I run compilation, I see this warning via VisualStudio:
- D:\DevOps\phyd\AEP.API.ESCONet.Customization.WebPortal\src\AEP.API.ESCONet.Customization.WebPortal.Services\LinterTest.cs(6,12,6,22): warning CS8618: Non-nullable field 'MioPrivato' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.
- warning CS8618: Non-nullable property '_mioPubblico' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.

and so on

**Comment by andreatosato** [comment]
[AEP.API.ESCONet.Customization.WebPortal.zip](https://github.com/github/super-linter/files/8939784/AEP.API.ESCONet.Customization.WebPortal.zip)

**Comment by andreatosato** [comment]
Why file is marked as empty? 

```txt
2022-06-20 09:12:02 [DEBUG]   Linting FILE: /tmp/lint/src/AEP.API.ESCONet.Customization.WebPortal/Program.cs
2022-06-20 09:12:02 [DEBUG]   FILE_STATUS (Program.cs) is empty, or not set to 'good' or 'bad'. Assuming it should be linted correctly. Setting FILE_STATUS to 'good'...
2022-06-20 09:12:02 [DEBUG]   File: /tmp/lint/src/AEP.API.ESCONet.Customization.WebPortal/Program.cs, FILE_NAME: Program.cs, DIR_NAME:/tmp/lint/src/AEP.API.ESCONet.Customization.WebPortal, FILE_STATUS: good, INDIVIDUAL_TEST_FOLDER: csharp, TEST_CASE_DIRECTORY: .automation/test/csharp
2022-06-20 09:12:02 [DEBUG]   CSHARP expects to lint individual files. Updated TEST_CASE_DIRECTORY to: .automation/test/csharp/
2022-06-20 09:12:02 [INFO]   ---------------------------
2022-06-20 09:12:02 [INFO]   File:[/tmp/lint/src/AEP.API.ESCONet.Customization.WebPortal/Program.cs]
2022-06-20 09:12:05 [INFO]    - File:[Program.cs] was linted with [dotnet-format] successfully
2022-06-20 09:12:05 [INFO]      - Command output:
------
  Formatting code files in workspace '/tmp/lint/src/AEP.API.ESCONet.Customization.WebPortal'.
  Format complete in 2706ms.
```
